pyproject.toml

```diff
@@ -49,7 +49,9 @@ all = [
     'pycountry>=23',
     'python-ulid>=1,<2; python_version<"3.9"',
     'python-ulid>=1,<3; python_version>="3.9"',
-    'pendulum>=3.0.0,<4.0.0'
+    'pendulum>=3.0.0,<4.0.0',
+    'pytz',
+    'tzdata',
```

Comment:
@07pepa can you lock these requirements, at least of what is compatible with `timezone_name` as versions